### PR TITLE
Posts cleanup

### PR DESCRIPTION
**Pull request by jburel**
Another round of cleaning

https://trello.com/c/DJXPIONO/26-news-tidy-up
cc @sbesson

### CODE REVIEW

File: _posts/2014-09-25-omero-bio-formats-5-0-5.md [hunk 1]
```diff
@@ -21,14 +21,10 @@ respective downloads page:
...
-For information on the upgrade from 4.4.x or 5.0.x to 5.0.5, see the OMERO5 instructions:
-
-[https://www.openmicroscopy.org/site/support/omero5.0/sysadmins/server-upgrade.html](https://www.openmicroscopy.org/site/support/omero5.0/sysadmins/server-upgrade.html)
+For information on the upgrade from 4.4.x or 5.0.x to 5.0.5, see the [OMERO 5 instructions](https://www.openmicroscopy.org/site/support/omero5.0/sysadmins/server-upgrade.html)
 
 For information on the upgrade from 4.4.x to 4.4.12, see the OMERO4 instructions, but upgrading
-directly to 5.0.5 would be preferred:
-
-[https://www.openmicroscopy.org/site/support/omero4/sysadmins/server-upgrade.html](https://www.openmicroscopy.org/site/support/omero4/sysadmins/server-upgrade.html)
+directly to 5.0.5 would be preferred, [see server upgrade](https://www.openmicroscopy.org/site/support/omero4/sysadmins/server-upgrade.html)
```
Comment (sbesson) [inline _posts/2014-09-25-omero-bio-formats-5-0-5.md]: `see [server upgrade]` maybe

File: _posts/2015-10-06-bio-formats-cpp-5-1-4.md [hunk 2]
```diff
@@ -3,11 +3,11 @@ layout: post
 title: Release of Bio-Formats-C++ 5.1.4
 intro-blurb: The OME team is pleased to announce the release of Bio-Formats-C++ 5.1.4 binary builds and ome-cmake-superbuild package
 ---
-Today we have released the binary builds of Bio-Formats-C++ 5.1.4, together with the ome-cmake-superbuild package used for building Bio-Formats-C++ and its dependencies on Windows and other platforms.  This is a new package split out from the Bio-Formats sources, which may be used to build the libraries required by Bio-Formats such as zlib, bzip2, icu, boost, libtiff and xerces-c.
+Today we have released the binary builds of Bio-Formats-C++ 5.1.4, together with the ome-cmake-superbuild package used for building Bio-Formats-C++ and its dependencies on Windows and other platforms.This is a new package split out from the Bio-Formats sources, which may be used to build the libraries required by Bio-Formats such as zlib, bzip2, icu, boost, libtiff and xerces-c.
```
Comment (sbesson) [inline _posts/2015-10-06-bio-formats-cpp-5-1-4.md]: Missing space `platforms.This`

File: _posts/2016-02-15-bio-formats-5-1-8.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title: Release of Bio-Formats 5.1.8
-intro-blurb: The OME Consortium are pleased to announce the release of Bio-Formats 5.1.8
+intro-blurb: The OME team are pleased to announce the release of Bio-Formats 5.1.8
```
Comment (sbesson) [inline _posts/2016-02-15-bio-formats-5-1-8.md]: is pleased

File: _posts/2016-02-18-bio-formats-cpp-5.1.8.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title: Release of Bio-Formats-C++ 5.1.8
-intro-blurb: The OME Consortium are pleased to announce the release of Bio-Formats-C++ 5.1.8
+intro-blurb: The OME team are pleased to announce the release of Bio-Formats-C++ 5.1.8
```
Comment (sbesson) [inline _posts/2016-02-18-bio-formats-cpp-5.1.8.md]: is pleased

File: _posts/2016-04-26-ome-files-cpp-0-1-1.md [hunk 5]
```diff
@@ -1,11 +1,9 @@
 ---
 layout: post
-title: Release of OME Files C++ 0.1.1
-intro-blurb: The OME Consortium are pleased to announce the release of OME Files C++ 0.1.1
+title: Release of OME-Files-C++ 0.1.1
+intro-blurb: The OME team are pleased to announce the release of OME-Files-C++ 0.1.1
```
Comment (sbesson) [inline _posts/2016-04-26-ome-files-cpp-0-1-1.md]: is pleased

File: _posts/2016-05-09-bio-formats-5-1-10.md [hunk 6]
```diff
@@ -1,21 +1,17 @@
 ---
 layout: post
 title: Release of Bio-Formats 5.1.10
-intro-blurb: The OME Consortium are pleased to announce the release of Bio-Formats 5.1.10
+intro-blurb: The OME team are pleased to announce the release of Bio-Formats 5.1.10
```
Comment (sbesson) [inline _posts/2016-05-09-bio-formats-5-1-10.md]: is pleased

File: _posts/2016-05-10-omero-5-2-3.md [hunk 7]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title: Release of OMERO 5.2.3
-intro-blurb: The OME Consortium are pleased to announce the release of OMERO 5.2.3
+intro-blurb: The OME team are pleased to announce the release of OMERO 5.2.3
```
Comment (sbesson) [inline _posts/2016-05-10-omero-5-2-3.md]: is pleased

File: _posts/2016-08-30-jpeg-xr-compression-support-coming-to-bioformats.md [hunk 8]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title: JPEG XR compression support coming to Bio-Formats
-intro-blurb: The OME Consortium are delighted to announce that thanks to a collaboration between Glencoe Software and Carl Zeiss Microscopy GmbH, support for whole slide images compressed with the JPEG XR compression scheme is coming to Bio-Formats
+intro-blurb: The OME team are delighted to announce that thanks to a collaboration between Glencoe Software and Carl Zeiss Microscopy GmbH, support for whole slide images compressed with the JPEG XR compression scheme is coming to Bio-Formats
```
Comment (sbesson) [inline _posts/2016-08-30-jpeg-xr-compression-support-coming-to-bioformats.md]: is delighted

File: _posts/2016-04-11-ome-files-cpp-0-1-0.md [hunk 9]
```diff
@@ -16,7 +16,7 @@ Major changes in this release include:
 - Support for transparent model upgrades and downgrades when reading an OME-TIFF
 - Splitting of the codebase into a separate repository for each component
 
-OME Files C++ was previously named "Bio-Formats C++" and was part of
+OME-Files-C++ was previously named "Bio-Formats-C++" and was part of
```
Comment (sbesson) [inline _posts/2016-04-11-ome-files-cpp-0-1-0.md]: Now hyphens needed `OME Files C++` is the name used everywhere

File: _posts/2016-04-11-ome-files-cpp-0-1-0.md [hunk 10]
```diff
@@ -124,7 +121,7 @@ Super-Build 0.1.0
 - use the same git tag-based versioning scheme used for bioformats and the other C++ git repositories
 - building with VS2012 is now possible; VS2013 remains the supported version
 
-Package changes:
+*Package changes:*
```
Comment (sbesson) [inline _posts/2016-04-11-ome-files-cpp-0-1-0.md]: Remove trailing colon

File: _posts/2016-04-26-ome-files-cpp-0-1-1.md [hunk 11]
```diff
@@ -1,11 +1,9 @@
 ---
 layout: post
-title: Release of OME Files C++ 0.1.1
-intro-blurb: The OME Consortium are pleased to announce the release of OME Files C++ 0.1.1
+title: Release of OME-Files-C++ 0.1.1
```
Comment (sbesson) [inline _posts/2016-04-26-ome-files-cpp-0-1-1.md]: OME Files C++

File: _posts/2016-04-26-ome-files-cpp-0-1-1.md [hunk 12]
```diff
@@ -23,7 +21,7 @@ The software release is available at [archived downloads](http://downloads.openm
 
 This page includes links to the individual repositories and source releases, as well as the documentation and binary builds of the collection for several common platforms, including Windows, Linux and MacOS X.
 
-**Changes since ome-files-cpp 0.1.0**
+**Changes since OME-Files-C++ 0.1.0**
```
Comment (sbesson) [inline _posts/2016-04-26-ome-files-cpp-0-1-1.md]: OME Files C++

File: _posts/2016-08-01-ome-files-cpp-0-2-0.md [hunk 13]
```diff
@@ -1,10 +1,8 @@
 ---
 layout: post
-title: Release of OME Files C++ 0.2.0
-intro-blurb: The OME team is pleased to announce the release of OME Files C++ 0.2.0
+title: Release of OME-Files-C++ 0.2.0
```
Comment (sbesson) [inline _posts/2016-08-01-ome-files-cpp-0-2-0.md]: OME Files C++

File: _posts/2016-08-01-ome-files-cpp-0-2-0.md [hunk 14]
```diff
@@ -1,10 +1,8 @@
...
+intro-blurb: The OME team is pleased to announce the release of OME-Files-C++ 0.2.0
```
Comment (sbesson) [inline _posts/2016-08-01-ome-files-cpp-0-2-0.md]: OME Files C++

File: _posts/2016-08-01-ome-files-cpp-0-2-0.md [hunk 15]
```diff
@@ -34,15 +32,14 @@ Major changes in this release include:
...
-page](http://downloads.openmicroscopy.org/ome-files-cpp/0.2.0/).
+The software release is available at [archived downloads](http://downloads.openmicroscopy.org/ome-files-cpp/0.2.0/).
 
 This page includes links to the individual repositories and source
 releases, as well as the documentation and binary builds of the
 collection for several common platforms, including Windows, Linux and
 MacOS X.
 
-**Changes since ome-files-cpp 0.1.1**
+**Changes since OME-Files-C++ 0.1.1**
```
Comment (sbesson) [inline _posts/2016-08-01-ome-files-cpp-0-2-0.md]: OME Files C++

File: _posts/2016-09-19-ome-files-cpp-0-2-1.md [hunk 16]
```diff
@@ -1,19 +1,17 @@
 ---
 layout: post
-title: Release of OME Files C++ 0.2.1
-intro-blurb: The OME team is pleased to announce the release of OME Files C++ 0.2.1
+title: Release of OME-Files-C++ 0.2.1
```
Comment (sbesson) [inline _posts/2016-09-19-ome-files-cpp-0-2-1.md]: OME Files C++

File: _posts/2016-09-19-ome-files-cpp-0-2-1.md [hunk 17]
```diff
@@ -1,19 +1,17 @@
...
+intro-blurb: The OME team is pleased to announce the release of OME-Files-C++ 0.2.1
```
Comment (sbesson) [inline _posts/2016-09-19-ome-files-cpp-0-2-1.md]: OME Files C++

File: _posts/2016-09-19-ome-files-cpp-0-2-1.md [hunk 18]
```diff
@@ -22,11 +20,11 @@ Major changes in this release include:
 **Downloads**
 
 The software release is available at
-[http://downloads.openmicroscopy.org/ome-files-cpp/0.2.1/](http://downloads.openmicroscopy.org/ome-files-cpp/0.2.1/)
+[archived downloads](http://downloads.openmicroscopy.org/ome-files-cpp/0.2.1/)
 
 This page includes links to the individual repositories and source releases, as well as the documentation and binary builds of the collection for several common platforms, including Windows, Linux and MacOS X.
 
-**Changes since ome-files-cpp 0.2.0**
+**Changes since OME-Files-C++ 0.2.0**
```
Comment (sbesson) [inline _posts/2016-09-19-ome-files-cpp-0-2-1.md]: OME Files C++

File: _posts/2016-10-12-ome-files-cpp-0-2-2.md [hunk 19]
```diff
@@ -1,13 +1,11 @@
 ---
 layout: post
-title: Release of OME Files C++ 0.2.2
-intro-blurb: The OME team is pleased to announce the release of OME Files C++ 0.2.2
+title: Release of OME-Files-C++ 0.2.2
```
Comment (sbesson) [inline _posts/2016-10-12-ome-files-cpp-0-2-2.md]: OME Files C++

File: _posts/2016-10-12-ome-files-cpp-0-2-2.md [hunk 20]
```diff
@@ -1,13 +1,11 @@
...
+intro-blurb: The OME team is pleased to announce the release of OME-Files-C++ 0.2.2
```
Comment (sbesson) [inline _posts/2016-10-12-ome-files-cpp-0-2-2.md]: OME Files C++

File: _posts/2016-10-12-ome-files-cpp-0-2-2.md [hunk 21]
```diff
@@ -24,12 +22,12 @@ Major changes in this release include:
...
 **Downloads**
 
-The software is available at:
+The software is available at
 [archived downloads](http://downloads.openmicroscopy.org/ome-files-cpp/0.2.2/).
 
 This page includes links to the individual repositories and source releases, as well as the documentation and binary builds of the collection for several common platforms, including Windows, Linux and MacOS X.
 
-**Changes since ome-files-cpp 0.2.0**
+**Changes since OME-Files-C++ 0.2.0**
```
Comment (sbesson) [inline _posts/2016-10-12-ome-files-cpp-0-2-2.md]: OME Files C++

File: _posts/2016-11-01-ome-files-cpp-0-2-3.md [hunk 22]
```diff
@@ -1,11 +1,9 @@
 ---
 layout: post
-title: Release of OME Files C++ 0.2.3
-intro-blurb: The OME team is pleased to announce the release of OME Files C++ 0.2.3
+title: Release of OME-Files-C++ 0.2.3
```
Comment (sbesson) [inline _posts/2016-11-01-ome-files-cpp-0-2-3.md]: OME Files C++

File: _posts/2016-11-01-ome-files-cpp-0-2-3.md [hunk 23]
```diff
@@ -1,11 +1,9 @@
...
+intro-blurb: The OME team is pleased to announce the release of OME-Files-C++ 0.2.3
```
Comment (sbesson) [inline _posts/2016-11-01-ome-files-cpp-0-2-3.md]: OME Files C++

File: _posts/2016-11-01-ome-files-cpp-0-2-3.md [hunk 24]
```diff
@@ -19,10 +17,10 @@ Changes in this release include:
 
 **Downloads**
 
-The software release is available at [downloads.openmicroscopy.org/ome-files-cpp/0.2.3/](http://downloads.openmicroscopy.org/ome-files-cpp/0.2.3/).
+The software release is available at [archived downloads](http://downloads.openmicroscopy.org/ome-files-cpp/0.2.3/).
 This page includes links to the individual repositories and source releases, as well as the documentation and binary builds of the collection for several common platforms, including Windows, Linux and MacOS X.
 
-**Changes since ome-files-cpp 0.2.2**
+**Changes since OME-Files-C++ 0.2.2**
```
Comment (sbesson) [inline _posts/2016-11-01-ome-files-cpp-0-2-3.md]: OME Files C++

File: _posts/2017-02-14-ome-files-cpp-0-3-0.md [hunk 25]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title: Release of OME-Files-C++ 0.3.0
-intro-blurb: The OME team is pleased to announce the release of OME Files C++ 0.3.0
+intro-blurb: The OME team is pleased to announce the release of OME-Files-C++ 0.3.0
```
Comment (sbesson) [inline _posts/2017-02-14-ome-files-cpp-0-3-0.md]: OME Files C++

File: _posts/2017-02-14-ome-files-cpp-0-3-0.md [hunk 26]
```diff
@@ -40,10 +38,9 @@ The main change is the switch to C++11. No immediate source changes are strictly
 -  the deprecated headers will be removed for 0.4.0.
 
 
-Changes since ome-files-cpp 0.2.3
----------------------------------
+**Changes since OME-Files-C++ 0.2.3**
```
Comment (sbesson) [inline _posts/2017-02-14-ome-files-cpp-0-3-0.md]: OME Files C++

File: _posts/2017-02-21-ome-files-cpp-0-3-1.md [hunk 27]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
-title: Release of OME Files C++ 0.3.1
-intro-blurb: The OME team is pleased to announce the release of OME Files C++ 0.3.1
+title: Release of OME-Files-C++ 0.3.1
```
Comment (sbesson) [inline _posts/2017-02-21-ome-files-cpp-0-3-1.md]: OME Files C++

File: _posts/2017-02-21-ome-files-cpp-0-3-1.md [hunk 28]
```diff
@@ -1,7 +1,7 @@
...
+intro-blurb: The OME team is pleased to announce the release of OME-Files-C++ 0.3.1
```
Comment (sbesson) [inline _posts/2017-02-21-ome-files-cpp-0-3-1.md]: OME Files C++

File: _posts/2017-02-21-ome-files-cpp-0-3-1.md [hunk 29]
```diff
@@ -13,15 +13,15 @@ This release contains two bugfixes to correct the support for references in the
...
+The software release is available at [archived downloads](http://downloads.openmicroscopy.org/ome-files-cpp/0.3.1/).
 
 This page includes links to the individual repositories and source releases, as well as the documentation and binary builds of the collection for several common platforms, including Windows, Linux and MacOS X.
 
 **Upgrading from 0.3.0**
 
 No changes required.
 
-**Changes since ome-files-cpp 0.3.0**
+**Changes since OME-Files-C++ 0.3.0**
```
Comment (sbesson) [inline _posts/2017-02-21-ome-files-cpp-0-3-1.md]: OME Files C++

File: _posts/2017-02-28-ome-files-cpp-0-3-2.md [hunk 30]
```diff
@@ -1,9 +1,9 @@
 ---
 layout: post
-title: Release of OME Files C++ 0.3.2
-intro-blurb: The OME team is pleased to announce the release of OME Files C++ 0.3.2
+title: Release of OME-Files-C++ 0.3.2
```
Comment (sbesson) [inline _posts/2017-02-28-ome-files-cpp-0-3-2.md]: OME Files C++

File: _posts/2017-02-28-ome-files-cpp-0-3-2.md [hunk 31]
```diff
@@ -1,9 +1,9 @@
...
+intro-blurb: The OME team is pleased to announce the release of OME-Files-C++ 0.3.2
```
Comment (sbesson) [inline _posts/2017-02-28-ome-files-cpp-0-3-2.md]: OME Files C++

File: _posts/2017-02-28-ome-files-cpp-0-3-2.md [hunk 32]
```diff
@@ -13,15 +13,15 @@ This release contains a single bugfix for the Minimal TIFF writer to align its s
...
+The software release is available at [archived downloads](http://downloads.openmicroscopy.org/ome-files-cpp/0.3.2/).
 
 This page includes links to the individual repositories and source releases, as well as the documentation and binary builds of the collection for several common platforms, including Windows, Linux and MacOS X.
 
 **Upgrading from 0.3.1**
 
 No changes required.
 
-**Changes since ome-files-cpp 0.3.1**
+**Changes since OME-Files-C++ 0.3.1**
```
Comment (sbesson) [inline _posts/2017-02-28-ome-files-cpp-0-3-2.md]: OME Files C++

File: _posts/2017-07-04-ome-files-cpp-0-4-0.md [hunk 33]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
-title: Release of OME Files C++ 0.4.0
-intro-blurb: The OME team is pleased to announce the release of OME Files C++ 0.4.0
+title: Release of OME-Files-C++ 0.4.0
```
Comment (sbesson) [inline _posts/2017-07-04-ome-files-cpp-0-4-0.md]: OME Files C++

File: _posts/2017-07-04-ome-files-cpp-0-4-0.md [hunk 34]
```diff
@@ -1,7 +1,7 @@
...
+intro-blurb: The OME team is pleased to announce the release of OME-Files-C++ 0.4.0
```
Comment (sbesson) [inline _posts/2017-07-04-ome-files-cpp-0-4-0.md]: OME Files C++

File: _posts/2017-07-04-ome-files-cpp-0-4-0.md [hunk 35]
```diff
@@ -30,7 +29,7 @@ MacOS X.
 
 - No changes required.
 
-**Changes since ome-files-cpp 0.3.2**
+**Changes since OME-Files-C++ 0.3.2**
```
Comment (sbesson) [inline _posts/2017-07-04-ome-files-cpp-0-4-0.md]: OME Files C++

File: _posts/2014-08-06-omero-bio-formats-5-0-3.md [hunk 36]
```diff
@@ -49,16 +49,16 @@ For system administrators, changes include:
 
 Note that the generated Apache 2.2 config contains a Rewrite rule `(RewriteRule ^/?$ /omero/ [R])` which no longer works (404 error), possibly due to changes in the OMERO.web custom prefix auto-configuration. Removing the rule fixes things.
 
-Further details are available on https://trac.openmicroscopy.org.uk/ome/milestone/5.0.3
+Further details are available on [trac](https://trac.openmicroscopy.org.uk/ome/milestone/5.0.3)
```
Comment (sbesson) [inline _posts/2014-08-06-omero-bio-formats-5-0-3.md]: Trac

File: _posts/2014-09-02-omero-bio-formats-5-0-4.md [hunk 37]
```diff
@@ -17,21 +17,18 @@ Bio-Formats improvements include:
 -  support for PicoQuant .bin files (thanks to Ian Munro)
 
 
-Full details are available on the OMERO Github milestone[1] and the Bio-Formats Github milestone[2] pages.
+Full details are available on the [OMERO Github milestone](https://github.com/openmicroscopy/openmicroscopy/issues?q=milestone%3A5.0.4+is%3Aclosed) and the [Bio-Formats Github milestone](https://github.com/openmicroscopy/bioformats/issues?q=milestone%3A5.0.4+is%3Aclosed) pages.
```
Comment (sbesson) [inline _posts/2014-09-02-omero-bio-formats-5-0-4.md]: GitHub

File: _posts/2016-04-11-ome-files-cpp-0-1-0.md (hunk 9 above)
Comment (jburel) [inline _posts/2016-04-11-ome-files-cpp-0-1-0.md]: will fix that 
I was following an initial file with B-F (which was wrong)

**Pull request merged**